src/amo/pages/Addon/index.js

```diff
@@ -204,8 +204,8 @@ export class AddonBase extends React.Component {
       );
     }
 
-    if (addon && addon.ratings.text_count) {
-      const count = addon.ratings.text_count;
+    if (addon && addon.ratings.count) {
```

Comment:
should we check for addon.ratings here instead/first?

Reply:
Whoops, good catch. I guess `ratings` [can be empty](https://github.com/mozilla/addons-frontend/blob/e706171bcc8b4b8f8cc6cbfdf6f5d1b918b35740/src/core/types/addons.js#L102) (although I can't think of why). The reason Flow didn't catch it here is because this file hasn't been converted to Flow. Sigh.

Reply:
ahh right.. 

Yes, I know this one has been talked about (converting this to Flow). Maybe I can work on that next week